stack.yaml

```diff
@@ -1,6 +1,9 @@
 resolver: lts-6.5
-system-ghc: false
 install-ghc: true
+flags: {}
 packages:
 - '.'
+system-ghc: false
 extra-deps:
+- bytestring-arbitrary-0.1.0
+- text-1.2.2.1
```

Comment:
Any reason why you need text listed as `extra-deps` here? When a library is already present in a Stackage snapshot (which is the case for `text`) you don't need to list it here explicitly. It will only make builds less stable, paradoxically, as it will impose yet another constraint to the cabal solver 😉